frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -69,9 +69,10 @@ export default class AccountsApi extends ExplorerApi {
       }
 
       if (accountsBasicInfo.length === 0) {
-        return {
-          accountId,
-        } as AccountBasicInfo;
+        throw {
+          account_id: accountId,
+          status: 404,
+        };
```

Comment:
Meh, this looks when one subsystem (libraries) tries to "directly" communicate to error display message on another subsystem (page rendering). We should consider building some boundary here and don't expect that the message from here will directly go into the props of the page.

There is no action item right now, but I want us to be mindful of these concerns.